.gitignore

```diff
@@ -1,5 +1,7 @@
 # See https://help.github.com/ignore-files/ for more about ignoring files.
 
+.yalc
```

Comment:
good candidate for `~/.gitignore_global` :slightly_smiling_face:

Reply:
i think we'll leave it in .gitignore in fact, because yalc is recommended in our README and is also used in the CI and not ignoring them will create problem, typically with checking there is no git diff / using with jest.